treeherder/webapp/api/resultset.py

```diff
@@ -82,7 +86,10 @@ def list(self, request, project, jm):
         filter = UrlQueryFilter(filter_params)
 
         offset_id = int(filter.pop("id__lt", 0))
-        count = min(int(filter.pop("count", 10)), 1000)
+        count = int(filter.pop("count", 10))
+
+        if count > MAX_RESULTS_COUNT:
+            return Response({"error": "Specified count exceeds API MAX_RESULTS_COUNT value: " + str(MAX_RESULTS_COUNT)}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
same comment as for the other, similar, PR:

    return Response(
        {"error": "Specified count exceeds API MAX_RESULTS_COUNT value: {}".format(MAX_RESULTS_COUNT)},
        status=status.HTTP_400_BAD_REQUEST
    )